### PR TITLE
create blog post on project unicorn learnings

### PR DESCRIPTION
**Pull request by roymoran**

### CODE REVIEW

File: content/blog/2019-09-03-project-unicorn-learnings/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: 'Project Unicorn Learnings'
+author: rmoran
+date: '2019-08-30'
+image: ./hero.jpg
+tags:
+  - Project Unicorn
+---
+
+After several months of working with developers and shipping software project on Project Unicorn - we've learned a few things when it comes to writing and building software with teams that form spontaneiously online. During this time, I’ve improved my software skills, picked up new technologies, practiced leading a small team of developers, and now have a clearer picture of what it takes to build and ship software with a small group of developers. Admittedly, it hasn't been easy so let me share some of the challenges we faced early on as a team. These are the same challenges many teams face early on when it comes to build software. The focus here less on technical challenges since those are less likely to derail a software team, but more on the challenges when it comes to momentum, planning, accountability, and seeking help.
```
Comment (rmjordas) [inline content/blog/2019-09-03-project-unicorn-learnings/index.md]: spontaneiously -> spontaneously

File: content/blog/2019-09-03-project-unicorn-learnings/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
... a slack/discord team, planning is extremely important. We’ve seen members immediately begin coding and those projects have failed. Here’s how that plays out. They lone wolf it for the first 1-2 weeks then realize they don’t know what they are building. By that time their team has lost momentum. Avoid these urges by planning first. Agree with your teammates on a set of core features, decide on the language/tech you will use, what role will each member will take on, and move forward from there. Break down what features are key for version 1 and drop everything else. This may take 1-2 weeks to do, but trust that this will work out better for your team in the long run. There are a couple practical suggestions here when it comes to planning your features. Organize your project into [user stories](https://www.atlassian.com/agile/project-management/user-stories) and create a [product backlog](https://www.atlassian.com/agile/scrum/backlogs). You can try the [moscow method](https://en.wikipedia.org/wiki/MoSCoW_method) to prioritize your features. If you're building a brand new product, ideally you want to boil down your users stories into a feature-set that takes 2 months or less to build.
```
Comment (rmjordas) [inline content/blog/2019-09-03-project-unicorn-learnings/index.md]: - slack -> Slack
- discord -> Discord
- moscow -> MoSCoW

File: content/blog/2019-09-03-project-unicorn-learnings/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
...ss 1 or more person on the project makes the effort to assign tasks, then nothing will get done. Progress will slowly stagnate. Eventually everyone loses steam and it’s over. So care enough to track your work. We tracked our work on a GitHub project board. And had weekly check-in calls at the end of the week to talk about what we worked on that week and what we will work on the upcoming week. Here’s an example of our [GitHub board](https://github.com/orgs/projectunic0rn/projects/3?fullscreen=true). For the calls we just hopped on a google hangout that lasted 30 minutes to an hour. I recapped our call on the slack group so everyone knew what they were responsible for that week. As developers sometimes we are afraid to be direct and tell others what they are responsible for. This is one of the pieces that separates an individual contributor from a technical leader. If no one else on your team is doing this take it as an opportunity and step in to drive your project. It can be intimidating at times especially when you operate under the misplaced notion that a lead developer should know more than the rest of the developers on the team. Drop that notion and step up to lead your project.
```
Comment (rmjordas) [inline content/blog/2019-09-03-project-unicorn-learnings/index.md]: - google hangout -> Google Hangouts
- slack -> Slack

File: content/blog/2019-09-03-project-unicorn-learnings/index.md (hunk 1 above)
Comment (DamonClark) [reply content/blog/2019-09-03-project-unicorn-learnings/index.md]: build -> building

Comment (aaron2212) [reply content/blog/2019-09-03-project-unicorn-learnings/index.md]: `software project` -> `software projects`
`the focus here less` -> `the focus here is less`

File: content/blog/2019-09-03-project-unicorn-learnings/index.md (hunk 2 above)
Comment (aaron2212) [reply content/blog/2019-09-03-project-unicorn-learnings/index.md]: `what role will each member will take o` -> `what role each member will take on`

File: content/blog/2019-09-03-project-unicorn-learnings/index.md (hunk 3 above)
Comment (aaron2212) [reply content/blog/2019-09-03-project-unicorn-learnings/index.md]: `1 or more person` -> `1 or more people`

File: content/blog/2019-09-03-project-unicorn-learnings/index.md (hunk 1 above)
Comment (roymoran) [reply content/blog/2019-09-03-project-unicorn-learnings/index.md]: cool made the fixes!

**Pull request merged**